modules/ROOT/pages/create-instance-task.adoc

```diff
@@ -3,50 +3,58 @@ ifndef::env-site,env-github[]
+An example: 
 
-After setting up an API version for management, you repeat the process to create another instance. Attempting to set up the same version creates another instance of the version. You can create multiple instances and expand the version information to view details about them.
+* Create an instance of an API and configure it as a proxy. Apply caching policies to this proxy. 
 
-To create a API instance:
+* Then, create an API instance to manage directly as a basic endpoint. Apply throttling and security policies to this basic endpoint. 
 
-. Select *API Administration > Manage API* to create an API instance from Exchange.
+* After configuring an API version for management, repeat the process to create another API instance. Specifying the same version creates another API instance of the version. 
+
+* Finally, create multiple API instances and click the version information to view API details.
+
+The following procedure describes creating an API instance from Exchange, and configuring the API instance as an endpoint with proxy.
+
+== Create an API instance
+
+. From *API Administration*, click *Manage API*. 
+. Select *Manage API From Exchange*.
 +
-For example, to set up version 1.0 of an API named `jsonplaceholder`:
+To configure version 1.0 of an example API named `jsonplaceholder`:
 +
 image::manage-exchange-end-task.png[]
 +
-. From *API Name*, select the API (`jsonplaceholder`).
-+
-NOTE: OpenAPI Specification (OAS) 3.0 is currently not supported in API Manager. You must convert the API to RAML or OAS 2.0 to manage it using API Manager. For converting the specifications to OAS 2.0, you must use third-party tools, such as APImatic.
+. From *API name*, enter `jsonplaceholder`.
 +
-. From *API Version*, select version (`1.0`).
-. From *Asset Version*, select the Exchange asset version.
-. From *Managing Type*, choose *Basic Endpoint* or *Endpoint with Proxy*. 
+NOTE: OpenAPI Specification (OAS) 3.0 is supported, with the exception of the callback feature. To work around this issue, handle the callback outside of the Mule runtime engine domain or use an OAS 3.0 specification that does not use callbacks.
 +
-For this example, select *Endpoint With Proxy*.
-. In *Implementation URI*, enter the URL that hosts your web site or an arbitrary URL, such as the example shown (`+http://api.domain.com+`). 
+. From *API version*, select version (`1.0`).
+. From *Asset version*, select the Exchange asset version.
+. From *Managing type*, select *Endpoint with proxy*.
+. In *Implementation URI*, enter your web site URL or an arbitrary URL, such as `+http://api.domain.com+`. 
 +
 If you choose _Endpoint with Proxy_ and do not make an entry (such as accepting the default value), you cannot save your configuration.
```

Comment:
Doesn't this belong after the step in Line 35?

Reply:
Fixed.